xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/ConfigurableClass.xml

```diff
@@ -82,8 +82,10 @@ $xwiki.jsfx.use('js/xwiki/actionbuttons/actionButtons.js', true)
       &amp;&amp; $currentDoc != 'XWiki.XWikiPreferences')
   ##
     #set($globaladmin = false)
+    #set($visibility = 'space')
   #else
     #set($globaladmin = true)
+    #set($visibility = 'wiki')
```

Comment:
Why do we need 2 variables (globaladmin and visibility)?

So I would:
* Remove the $globaladmin variable
* Name the variable $currentVisibility so that it's more clear that it represents the current visibility to be displayed

wdyt?